### DISCUSSION TITLE
Blog post

### DISCUSSION THREAD

**Discussion by flhoest** [PartialVolume/shredos.x86_64#188] [General]
Hi @PartialVolume, I'm busy writing a blog post about ShredOS.
How can I reach your directly ? My Twitter account is @flhoest We can DM there if you are ok ...

**Comment by PartialVolume** [comment]
Sounds interesting, I've not been on Twitter for years, however you can DM me [here](https://matrix.to/#/@partialvolume:matrix.org) . Click on the link and install element on your phone/desktop.

**Comment by PartialVolume** [comment]
As soon as you have created a user I'll add you into the ShredOS room.
